apisix/runner/server/handle.py

```diff
@@ -16,7 +16,7 @@
 #
 
 import flatbuffers
-import apisix.runner.plugin.core as runner_plugin
+from apisix.runner.plugin.core import PluginProcess as RunnerPlugin
```

Comment:
Why not use the original name?

Reply:
Uniform style with other names ^_^

Reply:
```suggestion
from apisix.runner.plugin.core import PluginProcess as runner_plugin
```
would be more consistent

Reply:
OK updated